Formula/ruby.rb

```diff
@@ -120,10 +118,24 @@ def install
       # Drop in the new version.
       rg_in.install Dir[buildpath/"vendor_gem/lib/*"]
       bin.install buildpath/"vendor_gem/bin/gem" => "gem#{program_suffix}"
+      (libexec/"gembin").install buildpath/"vendor_gem/bin/bundle" => "bundle#{program_suffix}"
+      (libexec/"gembin").install_symlink "bundle#{program_suffix}" => "bundler#{program_suffix}"
     end
   end
 
   def post_install
+    # Since Gem ships Bundle we want to provide that full/expected installation
+    # but to do so we need to handle the case where someone has previously
+    # installed bundle manually via `gem install`.
+    rm_f %W[
+      #{HOMEBREW_PREFIX}/bin/bundle
```

Comment:
Might as well use `rubygems_bindir` instead of `HOMEBREW_PREFIX}/bin` here, for consistency.